src/Features/CSharp/Portable/InlineDeclaration/CSharpInlineDeclarationCodeFixProvider.cs

```diff
@@ -160,22 +170,30 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
             var declarationExpression = GetDeclarationExpression(
                 sourceText, identifier, newType, singleDeclarator ? null : declarator);
 
+            return (local, identifier, declarationExpression, invocationOrCreation);
+        }
+
+        private SyntaxNode ReplaceNode(SemanticModel semanticModel, SyntaxNode currentRoot, (ILocalSymbol, IdentifierNameSyntax, DeclarationExpressionSyntax, SyntaxNode) t, SyntaxNode currentNode)
```

Comment:
This is a very general name for a method :) Consider adding some doc comments or change the method to something like `ReplaceIdentifierWithInlineDeclaration`

Reply:
Missed this comment, but still applicable to current code. Will update to new name